compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -634,6 +634,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
     pub fn read_discriminant(
         &self,
         op: &OpTy<'tcx, M::PointerTag>,
+        is_relative: bool,
```

Comment:
How can it be a property *of the operation* whether the discriminant is relative or not? Shouldn't that be determined by the layout of the enum? *confused*

Reply:
Oh, this is not saying whether the discriminant is relative, but whether we should be reading the relative or 'absolute' (?) discriminant. The variable name is very confusing. Maybe `read_as_relative` or so? Also the doc comment should explain what 'relative' even means -- relative to what?